fpdf/fpdf.py

```diff
@@ -1038,6 +1046,18 @@ def set_line_width(self, width):
         if self.page > 0:
             self._out(f"{width * self.k:.2f} w")
 
+    def set_background(self, background):
```

Comment:
Maybe call those `self.page_background` and `set_page_background()` to make it more explicit what they do?